components/private/Shapes/src/MDCShapedShadowLayer.h

```diff
@@ -63,4 +63,10 @@
  */
 @property(nonatomic, strong, nullable) id<MDCShapeGenerating> shapeGenerator;
 
+/*
+ The created CAShapeLayer representing the generated shape layer for the implementing UIView
+ from the shapeGenerator.
```

Comment:
What is the implementing UIView expected to do with this shape layer? Document here.

Reply:
documented